AssociativeArraysExercise/01-word-tracker.js

```diff
@@ -0,0 +1,9 @@
+function solve([searchingWordsStr, ...words]) {
+    const searchingWords = searchingWordsStr.split(' ');
+
+    const repetitionsOfWords = words.reduce((obj, word) => ({...obj, [word]: (obj[word] || 0) + 1}), {});
```

Comment:
Here you are unnecessary copying the obj every iteration. I would prefer
```suggestion
    const repetitionsOfWords = words.reduce((obj, word) => {
      obj[word] = obj[word] ? obj[word] + 1 : 1
      return obj;
    }, {});
```